packages/tasks/src/solanaDepositAndCall.ts

```
@@ -29,15 +32,20 @@
   let paramTypes;
   try {
     paramTypes = JSON.parse(args.types);
   } catch (error: any) {
     throw new Error(`Invalid JSON in 'types' parameter: ${error.message}`);
   }
   const params = [paramTypes, args.values];
-  await client.solanaDepositAndCall({ amount, params, recipient });
+  const res = await client.solanaDepositAndCall({
+    amount: args.amount,
+    values,
+    types: JSON.parse(args.types),
```

ESLint

[error] Expected object keys to be in ascending order. 'types' should be before 'values'.

(sort-keys-fix/sort-keys-fix)